app/src/components/extensionLoader/extensionLoader.jsx

```diff
@@ -35,18 +36,31 @@ ExtensionLoader.defaultProps = {
   withPreloader: false,
 };
 
-export function ExtensionLoaderWrapper({ extension, withPreloader, ...componentProps }) {
-  return extension.component ? (
-    <extension.component {...componentProps} />
-  ) : (
-    <ExtensionLoader extension={extension} withPreloader={withPreloader} {...componentProps} />
+export function ExtensionLoaderWrapper({
+  extension,
+  withPreloader,
+  silentOnError,
+  ...componentProps
+}) {
+  return (
+    <ErrorBoundary
+      getFallback={silentOnError ? undefined : (error) => <ErrorMessage error={error} />}
+    >
+      {extension.component ? (
+        <extension.component {...componentProps} />
+      ) : (
+        <ExtensionLoader extension={extension} withPreloader={withPreloader} {...componentProps} />
+      )}
+    </ErrorBoundary>
   );
 }
 ExtensionLoaderWrapper.propTypes = {
   extension: extensionType,
   withPreloader: PropTypes.bool,
+  silentOnError: PropTypes.bool,
 };
 ExtensionLoaderWrapper.defaultProps = {
   extension: {},
   withPreloader: false,
+  silentOnError: true,
```

Comment:
Maybe it is better to make it `false` by default?
Looks like we are using this component only with `silentOnError={false}`

Reply:
There are much more cases where I left it `true` as default (quality gate run info, unique errors page extensions, sidebar icons, etc.)